SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -201,8 +204,8 @@ Plot {
 				Pen.beginPath;
 				Pen.strokeColor = plotColor.at(0);
 				Pen.fillColor= plotColor.at(0);
-				this.perform(mode, xcoord, ycoord);
-				if (this.needsPenFill(mode)) {
+				this.perform(mode.at(0), xcoord, ycoord);
```

Comment:
`mode.at(0)` seems not correct if the mode of each plot is different.

Reply:
Yes, the new feature here is to add the ability for multiple plot modes, as you can have multiple plot colors. When `Plotter` is not `superpose`d, each `Plot` has only one plot mode, whereas when `superpose` is `true` the plot mode is an array of modes. IIC, I've redesigned this. See the main thread...